### ISSUE TITLE
Validation issue on multiple file upload

### ISSUE THREAD

**Issue by muradmustafayev** [codeigniter4/CodeIgniter4#2175]
In validation `getFile()` returns **NULL** in **CI4\system\Validation\FileRules.php** if send multiple file to validate. Should check if multiple or single upload
Same issue on other validation methods where is used `getFile()`

```
public function max_size(string $blank = null, string $params, array $data): bool
	{
		// Grab the file name off the top of the $params
		// after we split it.
		$params = explode(',', $params);
		$name   = array_shift($params);
		$file = $this->request->getFile($name); // returns NULL but files exist in $this->request. Should check if multiple or single upload
		if (is_null($file))
		{
			return false;
		}
		return $params[0] >= $file->getSize() / 1024;
	}
```

My Controller method
```
public function upload()
    {
        $validation =  \Config\Services::validation();
        $validation->setRules([
                'avatars' => 'max_size[avatars,2048]|mime_in[avatars,image/jpeg]|ext_in[avatars,jpeg,jpg]',
            ]
        );

        $validation->withRequest($this->request)->run();
        dd($validation->getErrors());
    }
```
HTML code
```
<?= form_open_multipart('admin/post/upload') ?>
        <input type="file" name="avatars[]" multiple>
        <input type="submit">
    <?= form_close() ?>
```

**Comment by muradmustafayev** [comment]
Another annoying thing is empty spaces in validation rules. I mean:
`mime_in[avatars, image/jpeg, application/pdf]` - not working
`mime_in[avatars,image/jpeg,application/pdf]` - works

It'd be handy if remove spaces with str_replace or regex.
original code - `$params = explode(',', $params);` (file: **CI4\system\Validation\FileRules.php**)

**Comment by jim-parry** [comment]
Spaces are not allowed in validation rules strings- that is not a bug.
https://codeigniter4.github.io/userguide/libraries/validation.html#available-rules
If you feel strongly about this, bring it up as an issue on its own, or raise it on the CI4 feature requests subforum.

**Comment by jim-parry** [comment]
Closed by #2265
